test/integration/cli-test.coffee

```diff
@@ -4,6 +4,9 @@ express = require 'express'
 clone = require 'clone'
 bodyParser = require 'body-parser'
 fs = require 'fs'
+syncExec = require 'sync-exec'
```

Comment:
Beware: I tried what I wrote above and it doesn't work out of the box. It needs more wrapping code. `execSync` is present in node >= 0.12, but works in slightly different way than I expected.